_includes/manuals/nightly/3.6_upgrade.md

```diff
@@ -95,6 +95,12 @@ If the Ruby 2.5 module was previously enabled, it must be reset:
 dnf -y module reset ruby
 {% endhighlight %}
 
+Switch to the PostgreSQL 13 module:
```

Comment:
Just thinking out loud: we reset 2.5 above (by now we can probably drop that). Here we explicitly switch. Why the difference?

Reply:
`switch-to` is "rather" new (I think the early versions of dnf in EL8 didn't have that)